### PR TITLE
MOON-90 add tab component

### PR DESCRIPTION
**Pull request by Taybou**

### CODE REVIEW

File: src/components/Tab/Tab.jsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Tab.scss';
+import classnames from 'clsx';
+import {Typography} from '../Typography';
+
+export const tabSizes = ['small', 'default', 'big'];
```
Comment (ffffffelix) [inline src/components/Tab/Tab.jsx]: As I can see the mockup just provide the medium size

File: src/components/Tab/Tab.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+export const Tab = ({label, onClick, size, isReversed, isDisabled, icon, variant, color, className, isSelected, ...props}) => {
+    let typoWeight = 'default';
+    if (size === 'small') {
+        typoWeight = 'light';
+    }
+
+    if (size === 'big') {
+        typoWeight = 'semiBold';
+    }
```
Comment (ffffffelix) [inline src/components/Tab/Tab.jsx]: There is just one size, you don't need that

File: src/components/Tab/Tab.jsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+    return (
+        <button
```
Comment (ffffffelix) [inline src/components/Tab/Tab.jsx]: Why you don't use the `Button` component?

Comment (FBerthelot) [inline src/components/Tab/Tab.jsx]: Add the possibility to override the main element. See  @vindhya if you want help on this.

File: src/components/Tab/Tab.jsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Tab.scss';
```
Comment (FBerthelot) [inline src/components/Tab/Tab.jsx]: I guess the name of this component is TabItem. The Tab component for me is the combinaisons of all TabItem ?

File: src/components/Tab/Tab.jsx (hunk 3 above)
Comment (Taybou) [reply src/components/Tab/Tab.jsx]: what did you mean by override the main element?

I'll see with her :D 

Thanks

Comment (Taybou) [reply src/components/Tab/Tab.jsx]: I update to something more generic

File: src/components/Tab/Tab.jsx (hunk 2 above)
Comment (Taybou) [reply src/components/Tab/Tab.jsx]: updated

File: src/components/Tab/Tab.jsx (hunk 1 above)
Comment (Taybou) [reply src/components/Tab/Tab.jsx]: updated

File: src/components/Tab/Tab.scss [hunk 5]
```diff
@@ -0,0 +1,8 @@
+.tab {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
```
Comment (ffffffelix) [inline src/components/Tab/Tab.scss]: Use `flexRow_center` and `alignCenter` classes

**Pull request merged**